service/frontend/dcRedirectionPolicy.go

```diff
@@ -172,14 +178,16 @@ func (policy *SelectedAPIsForwardingRedirectionPolicy) getTargetClusterAndIsDoma
 		return policy.currentClusterName, false
 	}
 
-	call := yarpc.CallFromContext(ctx)
-	enforceDCRedirection := call.Header(common.EnforceDCRedirection)
-	if !policy.config.EnableDomainNotActiveAutoForwarding(domainEntry.GetInfo().Name) && enforceDCRedirection != "true" {
-		// do not do dc redirection if auto-forwarding dynamic config and EnforceDCRedirection context flag is not enabled
+	if !policy.config.EnableDomainNotActiveAutoForwarding(domainEntry.GetInfo().Name) {
+		// do not do dc redirection if auto-forwarding dynamicconfig is not enabled
 		return policy.currentClusterName, false
 	}
 
-	_, ok := selectedAPIsForwardingRedirectionPolicyWhitelistedAPIs[apiName]
+	if policy.allDomainAPIs {
```

Comment:
Is it possible to have a per domain config?

Reply:
that's a good idea.